resources/dicts/events/ceremonies/ceremony_traits.json

```diff
@@ -3,200 +3,432 @@
+        "benevolence",
+        "heart of gold",
+        "exemplar",
+        "justice"
     ],
     "bloodthirsty": [
         "ferocity",
         "strength",
         "skill",
-        "courage"
+        "courage",
+        "vigor",
+        "boldness",
+        "desire",
+        "determination",
+        "dignity",
+        "cunning",
+        "justice"
     ],
     "bold": [
         "boldness",
         "fearlessness",
         "valiance",
         "daring",
-        "bravery"
+        "bravery",
+        "decisiveness",
+        "leadership",
+        "dauntlessness",
+        "strong-will",
+        "integrity",
+        "morals",
+        "dignity",
+        "vulnerability",
+        "honesty",
+        "exemplar",
+        "justice"
     ],
     "calm": [
         "serenity",
         "tranquility",
         "forethought",
-        "patience"
+        "patience",
+        "level-headedness",
+        "insight",
+        "thoughtfulness",
+        "reliability",
+        "phlegmatism",
+        "composure",
+        "emotional maturity",
+        "observance"
     ],
     "careful": [
         "mindfulness",
         "intelligence",
         "initiative",
-        "thoughtfulness"
+        "thoughtfulness",
+        "insight",
+        "forethought",
+        "stealth",
+        "pragmatism",
+        "wit",
+        "observance"
     ],
     "charismatic": [
         "warmth",
         "compassion",
         "kindness",
-        "energy"
+        "energy",
+        "charism",
+        "gregariousness",
+        "sociability",
+        "affection",
+        "openness",
+        "exemplar",
+        "wit"
     ],
     "childish": [
         "energy",
         "resilience",
         "spirit",
-        "enthusiasm"
+        "enthusiasm",
+        "spirit",
+        "humor",
+        "self-respect",
+        "compassion",
+        "intrigue"
     ],
     "cold": [
         "honesty",
         "cleverness",
         "skill",
-        "intelligence"
+        "intelligence",
+        "truthfulness",
+        "pragmatism",
+        "level-headedness",
+        "dignity",
+        "phlegmatism",
+        "composure",
+        "wit",
+        "observance"
     ],
     "compassionate": [
         "compassion",
         "kindness",
         "warmth",
-        "generosity"
+        "generosity",
+        "altruism",
+        "empathy",
+        "selflessness",
+        "vulnerability",
+        "exemplar",
+        "emotional maturity",
+        "justice",
+        "observance"
     ],
     "confident": [
         "confidence",
         "resilience",
         "boldness",
-        "initiative"
+        "initiative",
+        "exemplar",
+        "adaptability",
+        "dignity",
+        "independence"
     ],
     "daring": [
         "boldness",
         "fearlessness",
         "valiance",
         "daring",
-        "bravery"
+        "bravery",
+        "vigor",
+        "selflessness",
+        "intrigue",
+        "remedial actions",
```

Comment:
maybe I'm just not used to the phrase, but 'remedial actions' sounds a little too formal to me